maintainers/maintainer-list.nix

```diff
@@ -14625,4 +14625,10 @@
     github = "snpschaaf";
     githubId = 105843013;
   };
+  nsnelson = {
+    email = "noah.snelson@protonmail.com";
+    name = "Noah Snelson";
+    github = "peeley";
+    githubId = 30942198;
+  };
```

Comment:
This should be done in a seperate commit.

Reply:
Removed myself and then re-added in a separate commit. Not sure if I should have just made a separate PR, first time contributing!